addon/search/jump-to-line.js

```diff
@@ -42,10 +36,9 @@
 
       var match;
       if (match = /^\s*([\+\-]?\d+)\s*\:\s*(\d+)\s*$/.exec(posStr)) {
-        cm.setCursor(Pos(interpretLine(cm, match[1]), Number(match[2])))
+        cm.setCursor(interpretLine(cm, match[1]), Number(match[2]))
       } else if (match = /^\s*([\+\-]?\d+(\.\d+)?)\%\s*/.exec(posStr)) {
-        var prc = parseFloat(match[1]);
-        var line = Math.round(cm.lineCount() * prc / 100);
+        var line = Math.round(cm.lineCount() * Number(match[1]) / 100);
```

Comment:
To be consistent with your preference to `Number` instead of `parse***` -- and to match the terseness of the surrounding code.